packages/ui/src/components/label-marker.tsx

```diff
@@ -1,16 +1,20 @@
 import { FC } from 'react'
 
+import { Button, Icon } from '@/components'
 import { cn } from '@utils/cn'
-import { ColorsEnum } from '@views/labels'
+import { ColorsEnum, ILabelType, LabelType } from '@views/labels'
```

Comment:
Can we find a way to avoid references to the `views` from within `components`? Ideally this should be one-way to make it clear that views bring components together into a cohesive interface, leaving components as independent building blocks. Perhaps should `LabelMarker` be a subcomponent of the Labels view?